docs/participant_endpoint_enums.md

```diff
@@ -0,0 +1,37 @@
+# Participant Endpoint Enums
+
+This is the list of endpoints that a PISP or DFSP must register with the swith to get callbacks from the `thirdparty-api-adapter` and `transaction-request-service`.
+
+These endpoints should be added to the enums of `central-services-shared`, and the seeds of `central-ledger`
+
+## PISP:
+
+- `THIRDPARTY_CALLBACL_URL_CONSENT_REQUEST_PUT`
```

Comment:
Besides, can I check whether using plural would be preferred since the path is `consentRequests`? perhaps changing this to `THIRDPARTY_CALLBACK_URL_CONSENT_REQUESTS_PUT`

Reply:
Yeah the plural/non plural is a bit of a pain. I'm mostly following the existing conventions we use in the rest of Mojaloop for these enums: https://github.com/mojaloop/central-services-shared/blob/master/src/enums/endpoints.js